apps/meeting/src/utils/api.ts

```diff
@@ -37,12 +40,14 @@ export async function fetchMeeting(
 }
 
 export function createGetAttendeeCallback(meetingId: string) {
-  return async (chimeAttendeeId: string, externalUserId?: string) => {
-    const attendeeUrl = `${BASE_URL}attendee?title=${encodeURIComponent(
-      meetingId
-    )}&attendee=${encodeURIComponent(chimeAttendeeId)}`;
-    const res = await fetch(attendeeUrl, {
-      method: 'GET'
+  return async (chimeAttendeeId: string): Promise<GetAttendeeResponse> => {
```

Comment:
Could you restructure this function and make it same as `fetchMeeting` and `endMeeting`?

Reply:
Done